store/auth.go

```diff
@@ -158,3 +160,47 @@ func ReadStoreToken() (*StoreToken, error) {
 
 	return &readStoreToken, nil
 }
+
+// RequestPackageAccessMacaroon requests a macaroon for accessing package data from the ubuntu store
+func RequestPackageAccessMacaroon() (string, error) {
+	emptyJsonData := "{}"
+	req, err := http.NewRequest("POST", myappsPackageAccessAPI, strings.NewReader(emptyJsonData))
+	req.Header.Set("accept", "application/json")
+	req.Header.Set("content-type", "application/json")
+	if err != nil {
+		return "", fmt.Errorf("cannot get package access macaroon from store: %v", err)
+	}
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	// check return code, error on 4xx and anything !200
```

Comment:
As we discussed online, I understand the server really returns no possible errors. So let's simplify this a bit:

if resp.StatusCode != 200 {
        return "", fmt.Errorf("cannot get package access macaroon: store server returned status %d", resp.StatusCode)
}